controllers/replicationdestination.go

```diff
@@ -56,9 +56,9 @@ func (r *VolumeSnapshotRestoreReconciler) CreateReplicationDestination(log logr.
 func (r *VolumeSnapshotRestoreReconciler) buildReplicationDestination(replicationDestination *volsyncv1alpha1.ReplicationDestination, vsr *volsnapmoverv1alpha1.VolumeSnapshotRestore) error {
 
 	// get restic secret created by controller
-	resticSecretName := fmt.Sprintf("%s-secret", vsr.Name)
+	dmresticSecretName := fmt.Sprintf("%s-secret", vsr.Name)
```

Comment:
I believe the restic secret name is incorrect here.

Reply:
updated